### PR TITLE
Use encrypted service account JSON for uploading chart

### PR DESCRIPTION
**Pull request by MHBauer**
- Install gcloud for push to helm chart GCS bucket

### CODE REVIEW

**Comment by MHBauer** [comment]
I broke the push to vics copy of the repo, so now I'm just trying to see if this works.

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,9 @@ stages:
   - test
   - name: deploy
     if: type != pull_request
+  - name: push-chart
+    # require the tag name to match a regular expression
+    if: tag =~ ^v
```
Comment (MHBauer) [inline .travis.yml]: should tag be `$TRAVIS_TAG` ?

**Comment by MHBauer** [comment]
seems to have gotten past decrypt. I'll move it to right before calling the sync-script and we can test next release.

File: .travis.yml (hunk 1 above)
Comment (carolynvs) [reply .travis.yml]: The conditional stages/jobs have their own syntax

https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

File: .travis.yml [hunk 2]
```diff
@@ -31,6 +34,9 @@ jobs:
     - stage: test
       script:
       - |
+        # test to see if decryption works
```
Comment (carolynvs) [inline .travis.yml]: I'm glad you got it working! Still not quite sure what I did wrong the first time but hopefully I won't have to do it again. 😀

File: .travis.yml (hunk 1 above)
Comment (MHBauer) [reply .travis.yml]: cool. didnt know

**Comment by MHBauer** [comment]
I don't think there's anything else to do here but merge it and see what happens next release.

@viglesiasce can you check this?

**Comment by jeremyrickard** [comment]
LGTM

**Comment by MHBauer** [comment]
let's give it a shot for this weeks release then.

also, checking if I can: 
/approve

**Pull request merged**